src/nix_file.rs

```diff
@@ -210,6 +208,12 @@ impl NixFile {
             )
         };
 
+        if ast::Inherit::can_cast(attrpath_node.kind()) {
+            // Something like `inherit <attr>` or `inherit (<source>) <attr>`.
+            // We only look for ones like `<attr-path> = <value>`, so ignore this.
+            return Ok(Left(node.to_string()));
+        };
+
```

Comment:
```suggestion
        // This can only happen for Nix versions > 2.20, see the above `Attr` cast
        if ast::Inherit::can_cast(attrpath_node.kind()) {
            // Something like `inherit <attr>` or `inherit (<source>) <attr>`.
            // We only look for ones like `<attr-path> = <value>`, so ignore this.
            return Ok(Left(node.to_string()));
        };

```